docs/maintain-guides-how-to-monitor-your-node.md

```diff
@@ -0,0 +1,274 @@
+---
+id: maintain-guides-how-to-monitor-your-node
+title: Monitor your node
+sidebar_label: Monitor your node
+---
+
+This guide will walk you through how to set up [Prometheus](https://prometheus.io/) with
+[Grafana](https://grafana.com/) to monitor your node using Ubuntu 18.04.
+
+As Substrate-based chain would expose the data including the height of the chain, the number of
+connected peers to your node, CPU, and memory usage of your machine, etc. To monitor these data,
+Prometheus is capable of collecting metrics to allow Grafana for displaying it on the dashboard.
+
+## Preparation
+
+First, create a user for Prometheus and Node Exporter without the possibility to log in.
+
+```bash
+sudo useradd --no-create-home --shell /usr/sbin/nologin prometheus
+sudo useradd --no-create-home --shell /bin/false node_exporter
+```
+
+Create the directories required to store the configuration and executable files.
```

Comment:
Does Prometheus not do this when you install it? Just curious, because it's kinda surprising that it does not.

Reply:
good catch